README

```diff
@@ -4,26 +4,26 @@ This is version 10.0.0 of WiredTiger.
 
 WiredTiger release packages and documentation can be found at:
 
-	http://source.wiredtiger.com/
+	https://source.wiredtiger.com/
 
 The documentation for this specific release can be found at:
 
-	http://source.wiredtiger.com/10.0.0/index.html
+http://source.wiredtiger.com/10.0.0/index.html
```

Comment:
Can you change this one while we're here + put the indentation back to where it was?